docs/source/man.j2

```diff
@@ -0,0 +1,816 @@
+{{ QPC_VAR_PKG_SOURCE|default("qpc", true) }}
```

Comment:
This is valid for all variables:

I don't see why to set the default value each time this is called.
Why not simply set the default in the script (or make role) that will generate the man.rst?

The readability will be slightly improved and the maintenance will be GREATLY improved.

Reply:
(this is valid for the whole file and all variables)